File: beginners/easy/011.Immediate-odd-number/206.saadman-galib.immediateOddNumber.go
package main

import "fmt"

func main(){
	var n int 
	fmt.Scanln(&n)
	if n<=0{
		if n % 2 == 0{
			fmt.Println("Your immediate next odd number is =", n + 1)
		} else {
			fmt.Println("Invalid input")
		}
	} else{
		fmt.Println("You have entered a negative number")
	}
}

File: beginners/easy/011.Immediate-odd-number/206.saadman-galib.immediateOddNumber.cpp
#include<iostream>
using namespace std;

int main(){
    int num;
    cin >> num;
    if (num > 0){
        if (num %2 == 0){
            cout << "Your immediate next odd number is = " << num + 1 << endl;
        }
        else{
            cout << "Invalid input" << endl;
        }
    }
    else{
        cout << "You have to enter a positive number" << endl;
    }
    return 0;
}